Tests/UIKitTests/UIViewControllerExtensionsTests.swift

```diff
@@ -54,6 +54,18 @@ final class UIViewControllerExtensionsTests: XCTestCase {
         NotificationCenter.default.post(name: notificationIdentifier, object: nil)
         XCTAssertFalse(viewController.notificationFired)
     }
+    
+    class MyViewController: UIViewController {
+        var foo: String?
+    }
+    
+    func testInstantiate() {
+        let bar = "Bar"
+        let vc = MyViewController.instantiate(from: "TestStoryboard")
+        XCTAssertNotNil(vc)
```

Comment:
I'm not sure you can even get a `nil` result, so this test doesn't seem necessary.

Reply:
+1